projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -134,33 +134,38 @@ public function add_stats_menu() {
 
 	/**
 	 * Adds WordPress.com menu.
+	 *
+	 * @return number The current position of the submenu.
 	 */
 	public function add_wpcom_menu() {
-		if ( get_option( 'wpcom_is_staging_site' ) ) {
-			return;
-		}
+		add_menu_page( esc_attr__( 'WordPress.com', 'jetpack' ), __( 'WordPress.com', 'jetpack' ), 'publish_posts', 'wpcom', null, 'dashicons-wordpress-alt', 4 );
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blue.svg)

Sorry if I'm not fully up to date, but does this change imply that we're hiding only items related to Upgrades and displaying the following items? Is that correct?

```
WordPress.com
	Marketing
	Monetize
```